engine-tests/src/tests/eth_connector.rs

```diff
@@ -457,6 +457,100 @@ fn test_ft_transfer_call_eth() {
     assert_eq!(balance, transfer_amount);
 }
 
+#[test]
+fn test_ft_transfer_call_without_message() {
+    let (master_account, contract) = init(CUSTODIAN_ADDRESS);
+    call_deposit_eth_to_near(&contract, CONTRACT_ACC);
+
+    let balance = get_eth_on_near_balance(&master_account, DEPOSITED_RECIPIENT, CONTRACT_ACC);
+    assert_eq!(balance, DEPOSITED_AMOUNT - DEPOSITED_FEE);
+
+    let balance = get_eth_on_near_balance(&master_account, CONTRACT_ACC, CONTRACT_ACC);
+    assert_eq!(balance, DEPOSITED_FEE);
+
+    let res = contract.call(
+        CONTRACT_ACC.parse().unwrap(),
+        "register_relayer",
+        &RegisterRelayerCallArgs {
+            address: validate_eth_address(CUSTODIAN_ADDRESS),
+        }
+        .try_to_vec()
+        .unwrap(),
+        DEFAULT_GAS,
+        0,
+    );
+    res.assert_success();
+
+    let transfer_amount = 50;
+    // Send to Aurora contract with wrong message should failed
+    let res = contract.call(
+        CONTRACT_ACC.parse().unwrap(),
+        "ft_transfer_call",
+        json!({
+            "receiver_id": CONTRACT_ACC,
+            "amount": transfer_amount.to_string(),
+            "msg": "",
+        })
+        .to_string()
+        .as_bytes(),
+        DEFAULT_GAS,
+        1,
+    );
+    match res.outcome().status {
+        ExecutionStatus::Failure(_) => {}
+        _ => panic!("should panice"),
+    }
+
```

Comment:
Also please add a check that the balance hasn't been changed after the execution and transaction failure.

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/326/commits/28953c6f4b4bf43be5f5df56ba7bc778ec6877d4